### ISSUE TITLE
Queue menu bar extends beyond column

### ISSUE THREAD

**Issue by mrkylegp** [osTicket/osTicket#6157]
V1.16.2

When you have a larger number of queues, the "New ticket" button gets pushed beyond the column it is meant to stay in.

![Screenshot 2022-05-06 132835](https://user-images.githubusercontent.com/52687659/167062253-e708e900-a98d-4b79-92db-c73f32601a18.png)

**Comment by JediKev** [comment]
@mrkylegp 

I think this is because it's not factoring in the Top-Level Queue Counts. If you disable this setting it should show correctly.

Cheers.

**Comment by mrkylegp** [comment]
> @mrkylegp 
> 
> I think this is because it's not factoring in the Top-Level Queue Counts. If you disable this setting it should show correctly.
> 
> Cheers.

@JediKev yes, you are correct. Without the queue counts, it stays within It's boundaries.
